spec.html

```diff
@@ -8118,9 +8146,10 @@ <h1>Static Semantics: TopLevelVarScopedDeclarations</h1>
   <emu-clause id="sec-syntax-directed-operations-labels">
     <h1>Labels</h1>
 
-    <emu-clause id="sec-static-semantics-containsduplicatelabels" oldids="sec-statement-semantics-static-semantics-containsduplicatelabels,sec-block-static-semantics-containsduplicatelabels,sec-if-statement-static-semantics-containsduplicatelabels,sec-do-while-statement-static-semantics-containsduplicatelabels,sec-while-statement-static-semantics-containsduplicatelabels,sec-for-statement-static-semantics-containsduplicatelabels,sec-for-in-and-for-of-statements-static-semantics-containsduplicatelabels,sec-with-statement-static-semantics-containsduplicatelabels,sec-switch-statement-static-semantics-containsduplicatelabels,sec-labelled-statements-static-semantics-containsduplicatelabels,sec-try-statement-static-semantics-containsduplicatelabels,sec-function-definitions-static-semantics-containsduplicatelabels,sec-module-semantics-static-semantics-containsduplicatelabels" type="sdo" aoid="ContainsDuplicateLabels">
-      <h1>Static Semantics: ContainsDuplicateLabels</h1>
-      <p>With parameter _labelSet_.</p>
+    <emu-clause id="sec-static-semantics-containsduplicatelabels" oldids="sec-statement-semantics-static-semantics-containsduplicatelabels,sec-block-static-semantics-containsduplicatelabels,sec-if-statement-static-semantics-containsduplicatelabels,sec-do-while-statement-static-semantics-containsduplicatelabels,sec-while-statement-static-semantics-containsduplicatelabels,sec-for-statement-static-semantics-containsduplicatelabels,sec-for-in-and-for-of-statements-static-semantics-containsduplicatelabels,sec-with-statement-static-semantics-containsduplicatelabels,sec-switch-statement-static-semantics-containsduplicatelabels,sec-labelled-statements-static-semantics-containsduplicatelabels,sec-try-statement-static-semantics-containsduplicatelabels,sec-function-definitions-static-semantics-containsduplicatelabels,sec-module-semantics-static-semantics-containsduplicatelabels" type="sdo">
+      <h1>Static Semantics: ContainsDuplicateLabels ( _labelSet_ )</h1>
```

Comment:
I notice that some SDOs (e.g. EvaluateBody) have a 545-like header, with one parameter per line. But some (like this one), although they replace the "With parameter" paragraph with a parameter list in the `<h1>`, don't put it in 545 form. Is this form just transitional, draft-only?

Reply:
Ah, yeah, I didn't bother making them multiline when there were no types or optional parameters. But since the intent is to eventually list types for everything, I suppose we might as well make them multiline now.